### PR TITLE
Auditfix

### PR DESCRIPTION
**Pull request by zveinn**
## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
Fixing a counter variable placed in the wrong location and miss-reporting failed messages

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: internal/logger/target/http/http.go [hunk 1]
```diff
@@ -260,6 +257,7 @@ func (h *Target) send(ctx context.Context, payload []byte, payloadCount int, pay
 
 	resp, err := h.client.Do(req)
 	if err != nil {
+		h.failedMessages.Add(int64(payloadCount))
```
Comment (vadmeste) [inline internal/logger/target/http/http.go:260]: I think we can add this in error L240

**Review by vadmeste — COMMENTED** [review]
Can you rebase to fix the conflicts ?

File: internal/logger/target/http/http.go (hunk 1 above)
Comment (zveinn) [reply internal/logger/target/http/http.go:260]: done

Comment (vadmeste) [reply internal/logger/target/http/http.go:260]: I think you missed it, probably after a rebase

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**